### PR TITLE
Investigate removing no longer necessary dependencies while investigating  #2577

### LINKED ISSUE #2577: [Feature Request] Remove the indirectly dependency on system.drawing.common
**Is your feature request related to a problem? Please describe.**

We're using microsoft.identity.web.2.12.4 in one asp.net core web api project and the web api will be deployed in Kubernetes cluster environment.

microsoft.identity.web.2.12.4 has an indirect dependency on system.drawing.common.4.7.2. The dependency path is as below:

microsoft.identity.web.2.12.4.nupkg
↓
microsoft.identity.web.tokenacquisition.2.12.4.nupkg
↓
microsoft.identity.web.tokencache.2.12.4.nupkg
↓
system.drawing.common.4.7.2.nupkg

while per the description in "System.Drawing.Common only supported on Windows" (https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only), our team is required to remove any direct/indirect dependency on System.Drawing.Common library from our api project.

so we're investigating how to fix this issue.

**Describe the solution you'd like**
Provide a microsoft.identity.web version without any dependency on system.drawing.common

**Describe alternatives you've considered**
none

**Additional context**
none

### PR DESCRIPTION
**Pull request by jmprieur**
#Remove dependencies that are no longer necessary, while investigating  #2577
Remove the direct dependency on System.Windows.Drawing which was introduced due to a CVE more than one year ago,

Fixes #2577

### CODE REVIEW

**Comment by jmprieur** [comment]
The CVE-2021-24112 is still there in a dependency 
Investigating more

**Review by jennyf19 — APPROVED** [review]
:shipit:

File: Directory.Build.props [hunk 1]
```diff
@@ -86,9 +86,7 @@
     <MicrosoftGraphVersion>4.34.0</MicrosoftGraphVersion>
     <MicrosoftGraphBetaVersion>4.50.0-preview</MicrosoftGraphBetaVersion>
     <MicrosoftExtensionsHttpVersion>3.1.3</MicrosoftExtensionsHttpVersion>
-    <MicrosoftIdentityAbstractions>5.0.0</MicrosoftIdentityAbstractions>
-    <!--CVE-2021-24112-->
```
Comment (jmprieur) [inline Directory.Build.props:90]: Moved to where it applies that is only .NET Core 3.1

File: src/Microsoft.Identity.Web/Microsoft.Identity.Web.csproj [hunk 2]
```diff
@@ -22,7 +22,6 @@
     <PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="$(IdentityModelVersion)" />
     <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="$(IdentityModelVersion)" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="$(MicrosoftExtensionsHttpVersion)" />
-    <PackageReference Include="System.Drawing.Common" Version="$(SystemDrawingCommon)" />
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web/Microsoft.Identity.Web.csproj:25]: Not needed (transitive dependency of TokenCache)

File: tests/DevApps/WebAppCallsWebApiCallsGraph/Client/TodoListClient.csproj [hunk 3]
```diff
@@ -30,8 +30,6 @@
     <!--<PackageReference Include="Microsoft.AspNetCore.DataProtection.Abstractions" Version="3.1.1" />-->
     <PackageReference Include="Microsoft.Extensions.Caching.StackExchangeRedis" Version="5.0.1" />
     <PackageReference Include="StackExchange.Redis" Version="2.2.4" />
-    <!--CVE-2021-24112-->
```
Comment (jmprieur) [inline tests/DevApps/WebAppCallsWebApiCallsGraph/Client/TodoListClient.csproj:33]: Not needed (transitive dependency of TokenCache)

**Pull request merged**